web/source/dom/domManager.ts

```diff
@@ -1169,10 +1083,9 @@ namespace com.keyman.dom {
         Pelem._kmwAttachment.languageCode = Plc;
 
         // If Pelem is the focused element/active control, we should set the keyboard in place now.
-        // 'kmw_ip' is the touch-alias for the original page's control.
 
         var lastElem = this.lastActiveElement;
-        if(lastElem && (lastElem == Pelem || lastElem == Pelem['kmw_ip'])) {
+        if(lastElem && (lastElem == Pelem)) {
```

Comment:
Isn't `Pelem` guaranteed always non-null here?

Reply:
Actually... I don't see that it is, so I'll add a null guard earlier in the function.

That said, the lines in view of this comment... aren't a null check, so I'm not 100% sure what you were intending to get at here.

Reply:
`!this.isAttached(Pelem)` (line 1078) would be returning `false`, wouldn't it, if `Pelem === null`? Thus, `Pelem != null`. Then, that means if `lastElem == Pelem`, `lastElem` cannot be null... It's convoluted but I think it holds... Meaning the `lastElem [!=null]` clause is not needed.